RNTester/js/SwitchExample.js

```diff
@@ -11,10 +11,37 @@
 'use strict';
 
 const React = require('react');
-const ReactNative = require('react-native');
-const {Switch, Text, View} = ReactNative;
+const {Switch, Text, View} = require('react-native');
 
-class BasicSwitchExample extends React.Component<{}, $FlowFixMeState> {
+type OnOffIndicatorProps = $ReadOnly<{|on: boolean, testID: string|}>;
```

Comment:
I'm thinking I'll end up pulling these out to use on more tests but I don't want to overabstract until we know they are needed